src/modules/TaskClass.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] I have noticed that you are disabling linters in many files, please try not to do that because the purpose we use linters is learning to code by following the best practices. For example in some files you disabled default export, it is very easy to fix, and new line error as well. Kindly try to decrease the qty of `eslint-disable` 😃

Reply:
Line breakstyle comes up on Windows it comes up. But even if I remove it it won't cause any error on the liners.

Reply:
@IamYunusAli  I see, linters can be annoying sometimes! can you fix the default import export then ?